test/sw_mgmt_spec.rb

```diff
@@ -46,6 +46,24 @@
     }
   end
 
+  describe ".init" do
+    it "initializes package management and returns true" do
```

Comment:
Yes, the function itself is just a simple sequence of package management calls, no rocket science there...

The value is that the function is covered by tests (higher code coverage, that was actually my intent here) and we can at least ensure that there is no syntax error.

Or any idea how to improve it?